js/popup.js

```diff
@@ -0,0 +1,32 @@
+const languagesAPI = "https://translation.googleapis.com/language/translate/v2/languages?key=${key}";
+const languagesJSON = "../data/languages.json";
+
+document.addEventListener("DOMContentLoaded", () => {
+    loadSupportedLanguages();
+});
+
+/**
+ * Loads list of supported languages into a dropdown.
+ */
+function loadSupportedLanguages() {
+    fetch(languagesJSON)
+        .then(response => response.json()) 
+        .then(data => {
+            const dropdown = document.getElementById("languageDropdown");
+            const userLanguageCode = navigator.language;
+            const userLanguageOption = document.createElement('option');
+            userLanguageOption.value = userLanguageCode;
```

Comment:
just double checking that `createElement` doesnt reload the dom right? because that will call the observer in `scripts.js` that watches on page load

wouldnt it be nice to have test cases here? todo moment

Reply:
definitely an issue that needs to be opened up

Reply:
[@issue](https://github.com/wastadtlander/Youtube-Comment-Translator/issues/11)